server/database.js

```diff
@@ -42,15 +42,29 @@ function escapeRegExp(string) {
 const regexIgnoreDiacritics = (() => {
     const characterGroups = [
         ['[aàáâǎäãåā]'],
+        ['[bḃβ]'],
+        ['[cçćč]'],
+        ['[dďḋđδð]'],
         ['[eèéêëēėę]'],
+        ['[fƒḟ]'],
+        ['[gğģǧġĝǥγ]'],
+        ['[hḣĥħ]'],
         ['[iîïíīįì]'],
+        ['[jĵ]'],
+        ['[kķǩκ]'],
+        ['[lļľł₺λ]'],
+        ['[mṁμ]'],
+        ['[nñńŋňņṅⁿñν]'],
         ['[oôöòóøōõ]'],
+        ['[pṗ₽φψ]'],
+        ['[rŕřṙρ]'],
+        ['[sśşšșṡŝßσ]'],
+        ['[tţťțṫŧθτþ]'],
         ['[uûüùúū]'],
+        ['[wẇŵ₩]'],
+        ['[xẋξ]'],
         ['[yÿ]'],
-        ['[sśš]'],
-        ['[zžźż]'],
-        ['[cçćč]'],
-        ['[nñń]'],
+        ['[zžźżʒǯζ]'],
     ];
 
     const allCharacters = new RegExp(characterGroups.map(group => group[0]).join('|'), 'gi');
```

Comment:
This could be expanded upon later to include multi-character substitutions like "mu" or even "eszett"